app/javascript/app/services/api.js

```diff
@@ -0,0 +1,60 @@
+import isFunction from 'lodash/isFunction';
+import qs from 'query-string';
+
+const { CW_API_URL } = process.env;
+
+function handleResponse(d) {
+  if (d.status >= 200 && d.status <= 300) {
+    const data = isFunction(d.json) ? d.json() : d;
+    data.json = () => data;
+    return data;
+  }
+  throw new Error(d.statusText);
+}
+
```

Comment:
it's just for discussion: why not using `axios` for API calls for example?

Reply:
We have been using `fetch` trying to follow the standard but we could discuss it, going to the "concerns" doc